de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/Page.java

```diff
@@ -244,19 +244,17 @@ private void fetchByTitle(Title pTitle, boolean useExactTitle)
 	/**
 	 * @return Returns the id.
 	 */
-	protected long __getId()
+	long __getId()
```

Comment:
Is there a specific reason to downgrade this from protected to the default scope?